### ISSUE TITLE
[Bug] Cheers appear as invisible icons when adding them to my chat message after installing SevenTV

### ISSUE THREAD

**Issue by emongev** [SevenTV/Extension#255]
**Describe the bug**
When I add a cheer before SevenTV I get lets say a purple 100 with the spinning bit icon in my chat, but directly after installing SevenTV they appear like empty icons, so it appears like my  message is empty, which almost made me add multiple cheers accidentally.

**To Reproduce**

Before installing SevenTV
1. Go to the cheer icon
2. Click on 100
3. See how your chat message now has an animated emote representing the "Cheer100"

After installing SevenTV
1. Go to the cheer icon
2. Click on 100
3. See how your chat message appears to be empty.

**Comment by JaneOri** [comment]
## @ SevenTV devs:

This bit causes it:
https://github.com/SevenTV/Extension/blob/dev/src/Style/Style.scss#L75-L77

If that bit is necessary for some reason, add this css to Style.scss to fix it:
```css
    .chat-image-wrapper:has(img[alt*=Cheer]) {
      display: inline;
    }
```
doesn't work in firefox though: https://caniuse.com/css-has

---

## @ SevenTV users:

If anyone reading this has TamperMonkey installed, copy paste this into a new script to fix it:
```js
// ==UserScript==
// @name         fix cheer emotes in your text box when 7tv is installed
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  fix cheer emotes in your text box when 7tv is installed
// @author       JaneOri
// @match        https://www.twitch.tv/*
// @icon         https://www.google.com/s2/favicons?sz=64&domain=twitch.tv
// ==/UserScript==

setTimeout(() => document.head.appendChild(
  Object.assign(
    document.createElement("style"),
    { innerHTML: `.chat-image-wrapper:has(img[alt*=Cheer]) { display: inline; }` }
  )
), 1000)
```

**Comment by AnatoleAM** [comment]
There is no point reporting bugs on this version of this extension. It has been [completely rewritten](https://github.com/SevenTV/ExtensionV3) and 2.x.x will be going away soon, replaced by a wholly new extension.

**Comment by AnatoleAM** [comment]
Fixed in v3.0.0 (aaaac7f)
